modules/ROOT/pages/logging/security-log-analyzer.adoc

```diff
@@ -0,0 +1,170 @@
+:log-retention-days: 30
+:max-request-hours: 24
+
+[[aura-monitoring]]
+= Security log analyzer
+
+label:AuraDB-Professional[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+Security log analyzer is a feature that provides a UI to review the security events on an Aura instance.
+
+You can access security logs from *Logs* in the left-hand navigation and selecting *Security* in the tabs at the top.
+
+To toggle between instances, use the dropdown menu on the top left.
```

Comment:
"Toggle" to me sounds like picking between two options. Would prefer "select" or "switch".